site/src/docs/components/header/tabs.mdx

```diff
@@ -0,0 +1,35 @@
+---
+slug: '/components/header/tabs'
+title: 'Header'
+---
+
+import { StatusLabel, Notification } from 'hds-react';
+
+import LeadParagraph from '../../../components/LeadParagraph';
+import Layout from '../../../components/layout';
+import PageTabs from '../../../components/PageTabs';
+
+# Header
+
+<StatusLabel type="alert">WIP</StatusLabel>
+<StatusLabel type="success" style={{ marginLeft: 'var(--spacing-xs)' }}>
+  Accessible
+</StatusLabel>
+
+<LeadParagraph>
+  The header component is a wrapper for all navigation elements.
+</LeadParagraph>
+
+<Notification label="A work in progress!" className="siteNotification">
+  The header component is part of a collection of new navigation components that the HDS team is in the process of making. This means that this component is subject to change and we don't recommend using it in production just yet.
```

Comment:
Grammarly suggestion: "... component is part of a collection of new navigation components the HDS team is currently making. This means that this component is subject to change, and we don't recommend using it in production."